src/main/java/seedu/address/model/person/LessonSet.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.person;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.model.timetable.Lesson;
+
+/**
+ * This class was added to facilitate adding lessons to a person.
+ */
+public class LessonSet {
```

Comment:
Created to be attached to person, similar to ModuleTagSet

Reply:
Is this not going to be linked to ModuleTagSet?

Reply:
I think if this lesson set is initialised inside the ModuleTagSet, then it would be fine.

Reply:
Person does not need to know what lessons he has, according to our specification. So if we can uncouple them that would be good.